lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -3430,6 +3430,20 @@ and demonstrates generating options from a dataset directly.
 </param>
 ```
 
+The ``meta_file_key`` can be used together with ``from_dataset``. In such cases,
```

Comment:
Should we document that this works only with 21.01 / requires profile="21.01".

Reply:
Sure. Will do.